src/forge/production/lib/gradle/start/GradleStart.java

```diff
@@ -58,7 +58,7 @@ protected String getTweakClass() {
       protected void setDefaultArguments(Map<String, String> argumentMap) {
             argumentMap.put("version", "1.7.10");
             argumentMap.put("assetIndex", "1.7.10");
-            argumentMap.put("assetsDir", new File("").getAbsolutePath() + FILE_SEPERATOR + "lib" + FILE_SEPERATOR + "gradle" + FILE_SEPERATOR + "assets");
+            argumentMap.put("assestsDir", new File("").getAbsolutePath() + FILE_SEPERATOR + "Minecraft" + FILE_SEPERATOR + "production" + FILE_SEPERATOR + "lib" + FILE_SEPERATOR + "gradle" + FILE_SEPERATOR + "assests");
```

Comment:
Can you provide more information on the misspelling of "assets" here? If it's something that we've named, we should fix it. If it's coming from a third-party directory structure, it would seem likely to be corrected at some future release.

Reply:
My bad, that was just me misspelling assets it should be "assetsDir" and "assets"

Reply:
Ah, good. Correct that and I'll merge.